components/04-views/catalog/Category.stories.js

```diff
@@ -1,7 +1,7 @@
 import {storiesOf} from '@storybook/vue'
 
-import data from '../../../data/catalog.json'
-import menu from '../../../data/menu.json'
+import data from './mocks/catalog.json'
+import menu from '../../03-modules/header/mocks/header.json'
```

Comment:
the mock file name should describe what is inside, in this case menu, not the whole header